tests/src/test/java/alluxio/server/worker/WorkerAllMasterRegistrationTest.java

```diff
@@ -70,7 +71,6 @@ public void before() throws Exception {
         mNumMasters, mNumWorkers, PortCoordination.WORKER_ALL_MASTER_REGISTRATION);
     mCluster.initConfiguration(
         IntegrationTestUtils.getTestName(getClass().getSimpleName(), mTestName.getMethodName()));
-    Configuration.set(PropertyKey.MASTER_JOURNAL_CHECKPOINT_PERIOD_ENTRIES, 5);
```

Comment:
this line makes the master checkpoint more than it should be, and checkpointing logic is irrelevant to the tests here